.github/workflows/deploy.yml

```diff
@@ -0,0 +1,63 @@
+name: Deploy
+
+on:
+  workflow_dispatch:
+    inputs:
+      commit_hash:
+        description: commit hash to build and deploy
+        required: true
+
+jobs:
+  Build:
+    runs-on: windows-2022
+
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          ref: ${{ inputs.commit_hash }}
+
+      - name: Install Dependencies
+        shell: cmd
+        run: |
+          cd src
+          git clone --depth=1 https://github.com/ko4life-net/ko-vendor.git vendor
+
+      - name: Build Server
+        shell: cmd
+        run: |
+          call "C:\Program Files\Microsoft Visual Studio\2022\Enterprise\Common7\Tools\VsDevCmd.bat"
+          cd src
+          msbuild Server.sln -t:Clean -t:Build -p:Configuration=Release /m
+
+      - name: Upload to Server
+        uses: SamKirkland/FTP-Deploy-Action@4.3.3
+        with:
+          server: ${{ secrets.KO_SERVER_HOSTNAME }}
+          username: ${{ secrets.KO_SERVER_USERNAME }}
+          password: ${{ secrets.KO_SERVER_PASSWORD }}
+          local-dir: src/build/Release/
+          server-dir: build/Release/
```

Comment:
Maybe one last coment: Since we changed to use SSH key instead of password due to this comment: https://github.com/ko4life-net/ko/pull/181#discussion_r1070101096

Do you have the patience to change also this one to use SSH key instead? We could use the same PK for that.

We could technically run a simple `rsync` or `scp -r` command, but unfortunately I understand that `appleboy/ssh-action@v0.1.7` can only run on a Linux runner, whilst for building we use a Windows runner.

Good news is, that the same author of this action saying here: https://github.com/SamKirkland/FTP-Deploy-Action
"You must have ftp access to your server. If your host allows or requires ssh please use my [web-deploy](https://github.com/SamKirkland/web-deploy) action." So maybe we can change this to `web-deploy` instead? Example:
```yaml
- name: Upload to Server
  uses: SamKirkland/web-deploy@v1
  with:
    target-server: ${{ secrets.KO_SERVER_HOSTNAME }}
    remote-user: ${{ secrets.KO_SERVER_USERNAME }}
    private-ssh-key: ${{ secrets.KO_SERVER_PK }}
    source-path: src/build/Release/
    destination-path: build/Release/
```

Again, only if you have the patience testing this, otherwise let's just ship it 🚀 🚢

Reply:
For the recorded, it was implemented in the following commit: https://github.com/ko4life-net/ko/pull/181/commits/42098381f4177b2a9530753bd3975e52b2d8b552

🚀 💪